### PR TITLE
EPMDJ-4405: Update UI elements for scope action: delete

### PR DESCRIPTION
**Pull request by IliaSolovev**
Issue: EPMDJ-4405

### CODE REVIEW

**Review by RomanAchilov — APPROVED** [review]

File: src/pages/agent-full-page/coverage-plugin/scope/delete-scope-modal/delete-scope-modal.tsx [hunk 1]
```diff
@@ -71,16 +66,22 @@ export const DeleteScopeModal = deleteScopeModal(
...
+                      scope?.id && pathname.includes(scope.id)
+                        && push(`/full-page/${agentId}/${buildVersion}/${pluginId}/dashboard`);
                     },
                     onError: setErrorMessage,
                   })(scope as ActiveScope);
                 }}
+                data-test="delete-scope-modal:confirm-delete-button"
               >
-                {scope && scope.active ? 'Yes, Cancel Scope' : 'Yes, Delete Scope'}
+                {scope && 'Yes, Delete Scope'}
```
Comment (pavyarov) [inline src/pages/agent-full-page/coverage-plugin/scope/delete-scope-modal/delete-scope-modal.tsx:77]: emm? what?

Comment (pavyarov) [inline src/pages/agent-full-page/coverage-plugin/scope/delete-scope-modal/delete-scope-modal.tsx:77]:
```suggestion
                Yes, Delete Scope

```

File: src/pages/agent-full-page/coverage-plugin/scope/delete-scope-modal/delete-scope-modal.tsx [hunk 2]
```diff
@@ -71,16 +66,22 @@ export const DeleteScopeModal = deleteScopeModal(
...
               </DeleteScopeButton>
-              <Button type="secondary" size="large" onClick={() => onToggle(false)}>
+              <Button
+                type="secondary"
+                size="large"
+                onClick={() => onToggle(false)}
+                data-test="delete-scope-modal:cancel-modal-button"
```
Comment (pavyarov) [inline src/pages/agent-full-page/coverage-plugin/scope/delete-scope-modal/delete-scope-modal.tsx:83]: `data-test="delete-scope-modal:cancel-modal-button"` this is from kb?

File: src/pages/agent-full-page/coverage-plugin/scope/delete-scope-modal/delete-scope-modal.tsx [hunk 3]
```diff
@@ -28,22 +28,17 @@ export const DeleteScopeModal = deleteScopeModal(
...
-
     return (
       <Popup
         isOpen={isOpen}
         onToggle={onToggle}
         header={(
           <OverflowText>
-            {`${testsCount ? 'Delete' : 'Cancel'} Scope ${scope
-              && scope.name}`}
+            {`Delete Scope ${scope && scope.name}`}
```
Comment (pavyarov) [inline src/pages/agent-full-page/coverage-plugin/scope/delete-scope-modal/delete-scope-modal.tsx:41]: Now we can use optional chaining so:
```
{`Delete Scope ${scope?.name}`}
```

**Review by pavyarov — CHANGES_REQUESTED** [review]

File: src/pages/agent-full-page/coverage-plugin/scope/delete-scope-modal/delete-scope-modal.tsx (hunk 1 above)
Comment (IliaSolovev) [reply src/pages/agent-full-page/coverage-plugin/scope/delete-scope-modal/delete-scope-modal.tsx:77]: I replace it like you show me below

File: src/pages/agent-full-page/coverage-plugin/scope/delete-scope-modal/delete-scope-modal.tsx (hunk 2 above)
Comment (IliaSolovev) [reply src/pages/agent-full-page/coverage-plugin/scope/delete-scope-modal/delete-scope-modal.tsx:83]: In kb nothing about test-attrs but my task have so question

**Review by pavyarov — APPROVED** [review]

**Pull request merged**